[BUG] Bugfix 02000900 Compilation errors

### Did you test the latest `bugfix-2.0.x` code?

Yes, and the problem still exists.

### Bug Description

Had a series of errors while compiling under bugfix 02000900.  Compiles fine under version 02000801.  

Board is Chitu V8-190729, defined as BOARD_CHITU3D_V5.

Config files attached.

Errors list follows:
```
Marlin\src\lcd\tft_io\tft_io.cpp: In static member function 'static void TFT_IO::set_window(uint16_t, uint16_t, uint16_t, uint16_t)':
Marlin\src\lcd\tft_io\tft_io.cpp:189:19: error: 'ILI9341_CASET' was not declared in this scope
       io.WriteReg(ILI9341_CASET);
                   ^~~~~~~~~~~~~
Marlin\src\lcd\tft_io\tft_io.cpp:189:19: note: suggested alternative: 'ILI9488_CASET'
       io.WriteReg(ILI9341_CASET);
                   ^~~~~~~~~~~~~
                   ILI9488_CASET
Marlin\src\lcd\tft_io\tft_io.cpp:196:19: error: 'ILI9341_PASET' was not declared in this scope
       io.WriteReg(ILI9341_PASET);
                   ^~~~~~~~~~~~~
Marlin\src\lcd\tft_io\tft_io.cpp:196:19: note: suggested alternative: 'ILI9488_PASET'
       io.WriteReg(ILI9341_PASET);
                   ^~~~~~~~~~~~~
                   ILI9488_PASET
Marlin\src\lcd\tft_io\tft_io.cpp:203:19: error: 'ILI9341_RAMWR' was not declared in this scope
       io.WriteReg(ILI9341_RAMWR);
                   ^~~~~~~~~~~~~
Marlin\src\lcd\tft_io\tft_io.cpp:203:19: note: suggested alternative: 'ILI9488_RAMWR'
       io.WriteReg(ILI9341_RAMWR);
                   ^~~~~~~~~~~~~
                   ILI9488_RAMWR
Marlin\src\lcd\tft_io\tft_io.cpp:210:19: error: 'ILI9328_HASTART' was not declared in this scope
       io.WriteReg(ILI9328_HASTART);
                   ^~~~~~~~~~~~~~~
Marlin\src\lcd\tft_io\tft_io.cpp:210:19: note: suggested alternative: 'ILI9488_PASET'
       io.WriteReg(ILI9328_HASTART);
                   ^~~~~~~~~~~~~~~
                   ILI9488_PASET
Marlin\src\lcd\tft_io\tft_io.cpp:212:19: error: 'ILI9328_HAEND' was not declared in this scope
       io.WriteReg(ILI9328_HAEND);
                   ^~~~~~~~~~~~~
Marlin\src\lcd\tft_io\tft_io.cpp:212:19: note: suggested alternative: 'ILI9488_RAMRD'
       io.WriteReg(ILI9328_HAEND);
                   ^~~~~~~~~~~~~
                   ILI9488_RAMRD
```

### Bug Timeline

Problem only appears when compiling under bugfix 02000900.

### Expected behavior

_No response_

### Actual behavior

_No response_

### Steps to Reproduce

_No response_

### Version of Marlin Firmware

02000900

### Printer model

_No response_

### Electronics

Chitu V8-190729

### Add-ons

_No response_

### Your Slicer

_No response_

### Host Software

_No response_

### Additional information & file uploads

--- a/Marlin/src/lcd/tft_io/tft_io.cpp
+++ b/Marlin/src/lcd/tft_io/tft_io.cpp
@@ -39,18 +39,15 @@
 #if TFT_DRIVER == R61505 || TFT_DRIVER == AUTO
   #include "r65105.h"
 #endif
-#if TFT_DRIVER == ILI9328 || TFT_DRIVER == AUTO
-  #include "ili9328.h"
-#endif
-#if TFT_DRIVER == ILI9341 || TFT_DRIVER == AUTO
-  #include "ili9341.h"
-#endif
 #if TFT_DRIVER == ILI9488 || TFT_DRIVER == ILI9488_ID1 || TFT_DRIVER == AUTO
   #include "ili9488.h"
 #endif
 #if TFT_DRIVER == SSD1963 || TFT_DRIVER == AUTO
   #include "ssd1963.h"
 #endif
+
+#include "ili9341.h"
+#include "ili9328.h"
 
 #define DEBUG_OUT ENABLED(DEBUG_GRAPHICAL_TFT)
 #include "../../core/debug_out.h"

